Cubical/HITs/SetQuotients/Properties.agda

```diff
@@ -17,26 +17,147 @@ open import Cubical.HITs.SetTruncation
 open import Cubical.Core.Glue
 open import Cubical.Foundations.Equiv
 open import Cubical.Foundations.HLevels
+open import Cubical.Relation.Nullary
+open import Cubical.Relation.Binary.Base
+open import Cubical.Data.Nat.Base
+open import Cubical.Foundations.Univalence
+open import Cubical.Data.Nat.Properties
+open import Cubical.Foundations.HAEquiv
+
+
+Σ≡ : ∀ {ℓ ℓ'} {A : Set ℓ} {B : A → Set ℓ'}
```

Comment:
Some of the results here are already proved in https://github.com/agda/cubical/pull/83, so let's remove/move them to the proper file once that is merged

Reply:
Note that the equality for sigma-types we used is different : in #83 my definition is basically the one from the HoTT book, while this one makes use of the native dependent paths in CuTT.

Even though the HoTT-style equality can be useful at times, this one is basically a primitive so I think it is a better way of proving properties for sigma-types, and therefore that the proof of hLevelSigma below should be the one kept when merging.

Reply:
Interesting! I didn't realize that @loic-p used the HoTT book version. I would expect the one using native dependent paths to be a bit easier to use, so let's indeed use this one.